rllib/algorithms/impala/impala.py

```diff
@@ -874,17 +874,17 @@ def process_experiences_tree_aggregation(
     def update_workers_if_necessary(self) -> None:
         # Only need to update workers if there are remote workers.
         global_vars = {"timestep": self._counters[NUM_AGENT_STEPS_TRAINED]}
-        self._counters["steps_since_broadcast"] += 1
+        self._counters["num_training_step_calls_since_last_broadcast"] += 1
```

Comment:
Clarify name. "steps" is very confusing (env steps? agent steps?). We mean to count the number of times we have called `training_step()` here.

Reply:
this is a good idea. can we create a name for this string in rllib.utils.metrics?